### PR TITLE
Profiles CLI commands and messages improvements

### PR DESCRIPTION
**Pull request by RiccardoM**
## Description
This PR improves the DTag transfer related messages and CLI commands to make their names more explicit. 
It also changes the `blocklist` command to `blocks`

## Checklist
- [x] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.
- [ ] Wrote integration tests (simulation & CLI). 
- [x] Updated the documentation. 
- [x] Added an entry to the `CHANGELOG.md` file.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: x/profiles/types/msgs_dtag_requests_test.go [hunk 1]
```diff
@@ -179,13 +179,13 @@ func TestMsgRejectDTagRequest_Route(t *testing.T) {
 }
 
 func TestMsgRejectDTagRequest_Type(t *testing.T) {
```
Comment (dadamu) [inline x/profiles/types/msgs_dtag_requests_test.go:181]: The name of the message is changed to `MsgRefuseDTagTransfer` already. It should follow the name.

**Review by dadamu — CHANGES_REQUESTED** [review]
Overall looks good to me, just a minor thing about name of the test functions should be changed to new name.

File: proto/desmos/profiles/v1beta1/msgs_dtag_requests.proto [hunk 2]
```diff
@@ -89,5 +90,6 @@ message MsgRefuseDTagTransfer {
   string receiver = 2 [ (gogoproto.moretags) = "yaml:\"receiver\"" ];
 }
 
-// MsgRefuseDTagTransferResponse defines the Msg/RefuseDTagTransfer response.
-message MsgRefuseDTagTransferResponse {}
\ No newline at end of file
+// MsgRefuseDTagTransferRequestResponse defines the Msg/RefuseDTagTransfer
```
Comment (bragaz) [inline proto/desmos/profiles/v1beta1/msgs_dtag_requests.proto:93]:
```suggestion
// MsgRefuseDTagTransferRequestResponse defines the Msg/RefuseDTagTransferRequest
```

**Review by bragaz — COMMENTED** [review]
Some small fixes

**Pull request merged**